policyengine_us/parameters/gov/states/ne/dhhs/child_care_subsidy/special_need_age_threshold.yaml

```diff
@@ -0,0 +1,13 @@
+description: Nebraska limits the child care subsidy to children this age or younger, given they do require care due to a special need.  
+values:
+  2021-08-28: 19
+metadata:
+  period: year
+  unit: year
+  label: Nebraska child care subsidy special need child age threshold
+  reference:
+    # Age threshold not mentioned in legal code
```

Comment:
Does the legal code point to a different program?

Reply:
no oddly legal code only mentions income eligibility, none of the other parameters added (which I found in Nebraska DHHS sources)